libs/dotnet-sdk/FeatureBoardHttpClient.cs

```diff
@@ -59,6 +83,22 @@ public FeatureBoardHttpClient(HttpClient httpClient, LastETagProvider lastModifi
       _logger.LogDebug("Fetching updates done, eTag={eTag}", _eTag);
     }
 
+    RetryConditionHeaderValue? updateRetryAfterRef(RetryConditionHeaderValue? responseRetryAfter) // Sync method to allow use of eTag ref-local variable
+    {
+      ref var retryAfter = ref _retryAfter();
```

Comment:
After doing JS for ages, it's so weird being able to return a ref in this way. I'd just create a object which you update a property on it, but guess this avoids creating a class just to wrap the value